docs/blog/2019-03-27-security-concerns-unique-to-static-sites/index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Security for static sites.
```

Comment:
Gatsby is moving away from the term "static sites", since it isn't exactly true. Some alternatives could include:
- Security in Gatsby Sites
- Security with the Gatsby Website Framework
- Security for Modern Web Frameworks